src/api/routes/edition-group.js

```diff
@@ -183,15 +194,87 @@ router.get('/:bbid/identifiers',
  *             $ref: '#/definitions/Relationships'
  *       404:
  *         description: Edition Group not found
- *       406:
+ *       400:
  *         description: Invalid BBID
  */
 
 router.get('/:bbid/relationships',
-	makeEntityLoader('EditionGroup', relationshipsRelations, editionGroupError),
-	async (req, res) => {
+	makeEntityLoader('EditionGroup', utils.relationshipsRelations, editionGroupError),
+	async (req, res, next) => {
 		const editionGroupRelationshipList = await getEntityRelationships(res.locals.entity);
 		return res.status(200).send(editionGroupRelationshipList);
 	});
 
+/**
+ *	@swagger
+ * '/edition-group':
+ *   get:
+ *     tags:
+ *       - Browse Requests
+ *     summary: Gets a list of Edition-Group which are related to the other entity
+ *     description: BBID of an Edition is passed as a query parameter, and it's related EditionGroups are fetched
+ *     operationId: getRelatedPublisherByBbid
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: edition
+ *         in: query
+ *         description: BBID of the edition
+ *         required: true
+ *         type: bbid
+ *     responses:
```

Comment:
Another parameter is missing here:
```
name: type
in: query
description: filter by Edition Group type
type: string
enum: [book, leaflet, newspaper, magazine, journal]
```

I'm not sure how smart it is to have an enum when new elements can be added in the DB, but It might still be nicer to have it in the docs than leave it to the users to find out which types exist